### DISCUSSION TITLE
How to route traffic to different replicas which are scattered in different clusters?

### DISCUSSION THREAD

**Discussion by owenJiao** [karmada-io/karmada#2649] [Q&A]
If I use divided schedule type, which means the replicas of one pod will be scheduled to different clusters.  What confused me is how to route traffic to different replicas which are scattered in different clusters, I know in one cluster,  Service as a proxy  can route the external traffic to pods. But how does this feature work?

**Comment by GitHubxsy** [comment]
The following information may be useful：
https://karmada.io/docs/userguide/service/multi-cluster-service  
https://karmada.io/docs/userguide/service/multi-cluster-ingress
